### ISSUE TITLE
The extension doesn't open for any of the suggested file extensions

### ISSUE THREAD

**Issue by TomasHubelbauer** [excalidraw/excalidraw-vscode#91]
Hi, just installed the latest version of the Excelidraw extension and tried to use it by creating a `test.excelidraw`, `.excelidraw.json` and `test.excelidraw.png` files and opening them, both done from the VS Code Explorer pane. In the `.excelidraw` and `.excelidraw.json` cases, the files just opened as usual, as if there was no Excelidraw extension installed at all. In the `.excelidraw.png` case, I first got a hint that the PNG file was broken and could not be previewed and then VS Code asked me to fall back onto the default text editor, so very similar.

**Comment by abdukir** [comment]
I can say the same for my end as well. Just installed the extension to my latest VsCode here are my details. I don't know whats wrong with it, whenever i try to open any file related to the extension i get the "Failed to load Document: Error: Unable to load initial data" error

Version: 1.80.1 (user setup)
Commit: 74f6148eb9ea00507ec113ec51c489d6ffb4b771
Date: 2023-07-12T17:22:07.651Z
Electron: 22.3.14
ElectronBuildId: 21893604
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.19044

**Comment by stewartshea** [comment]
Ran into a similar issue with gitpod (gitpod code 1.8.0), which uses the openvsx extension registry. I also tried to download and install the extension via the Releases page and install through the vsix file, again with no luck

```
Gitpod Code
Version: 1.81.0
Commit: 598e5d080e7461be33923ee0d231bef718e5c65c
Date: 2023-08-03T19:07:27.589Z
Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36
```

Installed in my local vscode instance and it works fine; 

```
Visual Studio Code
Version: 1.79.2
Commit: 695af097c7bd098fbf017ce3ac85e09bbc5dda06
Date: 2023-06-14T08:59:55.818Z
Electron: 22.5.7
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Linux x64 6.2.0-26-generic
```

**Comment by micimize** [comment]
Just installed and I had to `Developer: Restart Extension Host` before the view would render properly

**Comment by bronius** [comment]
tldr; My case was unique bc I created a new, corrupted file in the way I have always worked in sublime/vscode:
1. cmd-n (opens new tab)
2. type the desired filename in the editor
3. hit save (file is created, extension tries to open the file with contents `filename.excalidraw` and fails

But opening the file in a more user-like way will work 🥇 

-------

> Just installed and I had to `Developer: Restart Extension Host` before the view would render properly

This was not my experience in local vscode with first-time attempted use of the extension in the way I first tried, but I found that it does work if I create a file from scratch-scratch-scratch. What I had done before (which was not working at all) was:

In VSCode, cmd-n to create a new file, named it, and save it in the desired folder:
- Create an empty file called `.excalidraw` as above, because I misread the instructions. Tried to open, got the error `Failed to load Document: Error: Unable to load initial data`
- Tried all the things (restart etc reinstall etc)
- Reread the instructions and renamed the file to something, didn't work
- Renamed to `super-unique-filenemememe.excalidraw` thinking may it was smth with the public access API, didn't work

What did work:
- In VSCode, right-click on a parent folder, click Create File, name it appropriately `newfile.excalidraw`
- And vscode automatically just opened the thing as expected beautifully

```
Identifier
pomdtr.excalidraw-editor
Version
3.8.1
Last Updated
2025-03-21, 13:21:10
```

```
Version: 1.98.2 (Universal)
Commit: ddc367ed5c8936efe395cffeec279b04ffd7db78
Date: 2025-03-12T13:32:45.399Z
Electron: 34.2.0
ElectronBuildId: 11161602
Chromium: 132.0.6834.196
Node.js: 20.18.2
V8: 13.2.152.36-electron.0
OS: Darwin arm64 24.3.0
```
